_posts/lazy-check-list-for-everything.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+header_image: /img/posts/lazy-check-list-for-everything/header.jpg
+title: "I’m lazy, I have a check list for everything"
+author: Mauro Servienti
+synopsis: "I don't want to waste brain cycles to remember things that aren't worth remembering. To-do lists and checklists are a much better approach. They help me not to forget things, and simultaneously, they free up my brainpower for more important matters."
+tags:
+- processes
+- self-organization
+- get-things-done
+---
+
+I'm turning 49 in a few days. I hate aging, but that's food for another story. When I was a kid, there were no digital devices of any sort. We got extremely excited when Sony released the [Walkman](https://en.wikipedia.org/wiki/Walkman). We were used to 45 RPM portable records players.
+
+In such a world, everyone used to memorize a lot of stuff. I still remember home and friends' phone numbers from 40 years ago. Adults used the prehistoric edition of a digital device, a paper "agenda" to remember things, but not kids. Using agendas was a terrible experience. I have clear in my mind my mother spending hours at the end of every year copying essential things, like phone numbers, from one planner to the new year's one.
+
+My digital device life started with a [Philips Velo 1](https://americanhistory.si.edu/collections/search/object/nmah_1827950), followed by a [Compaq iPAQ 3150](http://pencomputing.com/frames/compaq_ipaq_3610.html). Everything changed: no more need to remember phone numbers, no more need to have a paper planner, and more advantages.
+
+Nowadays, everything is different. Devices swamp us. The market offers every sort of option. From smartphones to smart rings, we can buy a device for every possible use. Many of them are indeed a solution in search of a problem, but that's another story.

```

Comment:
Made this suggestion because you said "that's another story" earlier

```suggestion
Nowadays, everything is different. Devices swamp us. The market offers every sort of option. From smartphones to smart rings, we can buy a device for every possible use. Many of them are indeed a solution in search of a problem, but that's also another story.
```